.eslintignore

```diff
@@ -62,3 +62,5 @@ site/public/blocks
 site/public/schemas
 site/site-map.json
 site/test-results
+packages/@blockprotocol/type-system-node/src
+packages/@blockprotocol/type-system-web/src
```

Comment:
We can copy this to `.prettierignore` too

Reply:
Updated the ignore files 
https://github.com/blockprotocol/blockprotocol/pull/503/commits/ac445d6c1b9193d089b951d4da434febc60011fd